crates/bevy_ecs/src/system/input.rs

```diff
@@ -11,6 +13,9 @@ use crate::{bundle::Bundle, prelude::Trigger, system::System};
 /// - [`InMut<T>`]: For mutable references to values
 /// - [`Trigger<E, B>`]: For [`ObserverSystem`]s
 /// - [`StaticSystemInput<I>`]: For arbitrary [`SystemInput`]s in generic contexts
+/// - Tuples of [`SystemInput`]s up to 8 elements
```

Comment:
This should also be documented on `In`.

Reply:
`In` used to be the *only* way to pass input into systems, but given it's now *just* one of the multiple `SystemInput` types, I don't think `In` needs to be treated any differently from the others. I have added an extra note on all of the input types to check the trait for more information.